### ISSUE TITLE
Invalid DOMTokenList.prototype.contains() in polyfill

### ISSUE THREAD

**Issue by josielrocha** [dmatteo/jsdomify#17]
Checking if element contains a substring of a given string as classname, DOMTokenList.prototype.contains() mistakenly returns true. Look below:

``` javascript
var p = document.createElement('p');
p.appendChild(document.createTextNode('Blah'));
document.body.appendChild(p);

p.classList.add('whooooo');
p.classList.contains('who') // Returns true
```

**Comment by dmatteo** [comment]
Hey @josielrocha which version of jsdomify are you using?

**Comment by josielrocha** [comment]
@dmatteo Latest, look here [https://github.com/podio/jsdomify/blob/master/src/polyfills/classList.js#L52](https://github.com/podio/jsdomify/blob/master/src/polyfills/classList.js#L52)

Perhaps .contains() method could be like:

``` javascript
contains: function(token) {
    // Old way
    // return this.el.className.indexOf(token) != -1;

    // I did use a RegExp here because someone could add a classname with spaces
    return this.el.className.split(/\s+/g).indexOf(token) !== -1;
},
```

**Comment by dmatteo** [comment]
Since we're now using jsdom 7.x, we can get rid of the polyfill entirely since classList has been introduced in 6.2.0 https://github.com/tmpvar/jsdom/pull/1213

**Comment by josielrocha** [comment]
@dmatteo I agree. Thanks.

**Comment by dmatteo** [comment]
@josielrocha Released as 1.0.1 thanks
